script/mafia

```diff
@@ -87,7 +87,7 @@ exec_mafia () {
 
         bin/bootstrap ) || exit $?
 
-      cp "$MAFIA_TEMP/mafia/.cabal-sandbox/bin/mafia" "$MAFIA_PATH_TEMP"
+      cp "$MAFIA_TEMP/mafia/dist/build/mafia/mafia" "$MAFIA_PATH_TEMP"
```

Comment:
there should be two directories "dist/build/mafia" and "dist/build/Mafia", not sure how case sensitive HFS+ works with that? with case insensitive it thinks they're the same and dumps all the files in there like a savage